src/Compilers/VisualBasic/Portable/Errors/Errors.vb

```diff
@@ -1760,6 +1760,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         ERR_BadAbstractStaticMemberAccess = 37314
         ERR_UnimplementedSharedMember = 37315
 
+        ERR_UnmanagedCallersOnlyNotSupported = 37316
```

Comment:
>ERR_UnmanagedCallersOnlyNotSupported [](http://example.com/codeflow?start=8&length=36)

Was this treated as a warning before?

Reply:
No, it was treated as an error. I created a local VB project and ran the modified test case to be sure, and they were treated as errors.